streamer/src/nonblocking/sendmmsg.rs

```diff
@@ -177,7 +177,13 @@ mod tests {
         ];
         let dest_refs: Vec<_> = vec![&ip4, &ip6, &ip4];
 
-        let sender = UdpSocket::bind("0.0.0.0:0").await.expect("bind");
+        let sender = bind_to_async(
```

Comment:
Given that we have a `bind_to_localhost_async()`, it wouldn't seem too crazy to add a `bind_to_unspecified_async()`. But, this test looks to contain the only two instances so I don't think this is a "must-do" either

Reply:
true. tbh probably worth it to add it for future development use if needed